config/config.py

```diff
@@ -1,187 +1,240 @@
-import yaml, os
+import yaml
+import os
 
-class _DeskConfigDAO:
-	"""
-	Desk Config DAO
-	"""
 
-	KEY_ID = "id"
-	KEY_NAME = "name"
-	KEY_SENSOR_PIN = "sensor_pin"
+class _SensorTypeConfigDAO:
+    """
+    Sensor Type Config DAO
+    """
 
-	def __init__(self, desk):
-		self.__desk_id = desk[self.KEY_ID]
-		self.__desk_name = desk[self.KEY_NAME]
-		self.__desk_pin = desk[self.KEY_SENSOR_PIN]
+    KEY_TYPE = "type"
+    KEY_NAME = "name"
 
-	@property
-	def id(self):
-		return self.__desk_id
+    def __init__(self, sensor):
+        self.__sensor_type = sensor[self.KEY_TYPE]
+        self.__name = sensor[self.KEY_NAME]
 
-	@property
-	def name(self):
-		return self.__desk_name
+    @property
+    def sensor_type(self):
+        return self.__sensor_type
 
-	@property
-	def pin(self):
-		return self.__desk_pin
+    @property
+    def sensor_name(self):
+        return self.__name
 
+    def __str__(self):
+        return "Sensor:\n\tType: " + str(self.__sensor_type) + "\n\tName: " + str(self.__name)
 
-class _DebugConfigDAO:
-	"""
-	Debug Config DAO
-	"""
 
-	KEY_ENABLED = "enabled"
-	KEY_REMOTE_SIMULATOR_CONFIG = "remote-simulator"
+class _DeskSensorConfigDAO:
+    """
+    Desk Sensor Config DAO
+    """
+
+    KEY_TYPE = "type"
+    KEY_BUS = "i2c_bus"
+    KEY_ADDRESS = "i2c_address"
+
+    def __init__(self, desk):
+        self.__sensor_type = desk[self.KEY_TYPE]
+        self.__i2c_bus = desk[self.KEY_BUS]
+        self.__i2c_address = desk[self.KEY_ADDRESS]
+
+    @property
+    def sensor_type(self):
+        return self.__sensor_type
+
+    @property
+    def i2c_bus(self):
+        return self.__i2c_bus
+
+    @property
+    def i2c_address(self):
+        return self.__i2c_address
+
+    def __str__(self):
+        return "Desk Sensor:\n\t\tType: " + str(self.__sensor_type) + "\n\t\ti2c_bus: " + str(self.__i2c_bus) + \
+               "\n\t\ti2c_address: " + hex(self.__i2c_address)
+
+
+class _DeskConfigDAO:
+    """
+    Desk Config DAO
+    """
+
+    KEY_ID = "id"
+    KEY_NAME = "name"
+    KEY_SENSOR = "sensor"
 
-	def __init__(self, debug):
-		self.__enabled = debug[self.KEY_ENABLED]
-		self.__remote_simulator_config = _RemoteSimulatorConfigDAO(debug[self.KEY_REMOTE_SIMULATOR_CONFIG])
+    def __init__(self, desk):
+        self.__desk_id = desk[self.KEY_ID]
+        self.__desk_name = desk[self.KEY_NAME]
+        self.__desk_sensor = _DeskSensorConfigDAO(desk[self.KEY_SENSOR])
```

Comment:
Why not send the whole desk to `_DeskSensorConfigDAO`? That way `DeskHandlerService` doesnt have to send redundant parameters